includes/Plugins.php

```diff
@@ -63,6 +63,11 @@ final class Plugins {
 				),
 			),
 			'hasYithExtended' => array(
+				array(
+					'slug'     => 'woocommerce',
+					'activate' => true,
+					'priority' => 300,
+				),
```

Comment:
Won't this be repetitive for ecom customers and Sitegen Customers??
I believe that the priority should be handling it right?

Reply:
That's okay, https://github.com/newfold-labs/wp-module-installer/blob/2f7b1a2d128703119bc5090e93e9cdd3045044b4/includes/TaskManagers/PluginInstallTaskManager.php#L154 will handle that.